### DISCUSSION TITLE
How to resolve missing import from com.netflix.graphql.dgs.client.codegen

### DISCUSSION THREAD

**Discussion by maines-pet** [Project-Books/books-api#110] [Q&A] [answered]
Hi,

After cloning the project locally, I've tried to run the project but I'm encountering package not found in the generated code as below:
![image](https://user-images.githubusercontent.com/21050686/131131918-3059c267-6fbe-4ea6-becc-76d94005190f.png)

I have tried running a gradle clean build but no luck.

Thanks in advance.

**Comment by knjk04** [comment, marked as answer]
Hi,

Thanks for bringing this up! I also had the same issue. I've fixed this by upgrading the codegen version to the latest. It's on the `main` branch, so try pulling and see if it works now

**Reply by keomony** [reply]
Hi,
I also tried to upgrade codegen to the latest but I got compile error and flyway initialiser error ( migration not working) 

![Screenshot 2021-08-29 at 19 52 27](https://user-images.githubusercontent.com/11650292/131262123-4b7f5c32-54fc-4ec5-93f6-c41e0b7dad3b.png)

**Reply by keomony** [reply]
Thanks in advance.

**Reply by knjk04** [reply]
Could you please copy the full error into a GitHub gist (or similar)?

**Reply by keomony** [reply]
Hi,
Yes here are links to the gists 

[Detected failed migration to version 010 (RENAME PUBLISHER BOOK CONSTRAINTS)](https://gist.github.com/keomony/8136d0ee198df1932ac36d536aeb3e9c)
[gradle flywayMigration](https://gist.github.com/keomony/a9360dc1fb56b10e30364003e86bf422)

**Reply by maines-pet** [reply]
@knjk04  thanks for the resolution, I somehow can't mark your answer

@keomony, I too have encountered that. What I did was to drop the schema/tables in MySQL then rerun.

**Reply by keomony** [reply]
@maines-pet : I still have the same errors. Could you tell me the steps you did? Thanks in advanced.

**Reply by maines-pet** [reply]
Here's what I did:

1. Removed the resources/db/migration/V010__RENAME_PUBLISHER_BOOK_CONSTRAINTS.sql as per #99 
2. Drop all the tables for the booksapi in MySQL
3. Restart the application
